commit b89ff577fcbd085fbd180f59aa669dded1818082
Author: Bernhard Beschow <bbeschow@cs.tu-berlin.de>
Date:   Wed Feb 8 18:14:59 2012 +0100

    compile with Qt < 4.8
    
    * take into account the comment in the API doc, which suggests to use QColor::rgb()
    (cherry picked from commit 6eafb31f509c622e2672c31b73ebcb1b3dafb3e9)

diff --git a/src/lib/TextureColorizer.cpp b/src/lib/TextureColorizer.cpp
index d0deead..9319a89 100644
--- a/src/lib/TextureColorizer.cpp
+++ b/src/lib/TextureColorizer.cpp
@@ -176,7 +176,7 @@ void TextureColorizer::colorize( QImage *origimg, const ViewportParams *viewport
         m_coastImage = QImage( viewport->size(), QImage::Format_RGB32 );
 
     // update coast image
-    m_coastImage.fill( QColor( 0, 0, 255, 0) );
+    m_coastImage.fill( QColor( 0, 0, 255, 0).rgb() );
 
     bool doClip = false; //assume false
     switch( viewport->projection() ) {
